scene/gui/control.cpp

```diff
@@ -752,16 +747,10 @@ bool Control::can_drop_data(const Point2 &p_point, const Variant &p_data) const
 		}
 	}
 
-	if (get_script_instance()) {
-		Variant v = p_point;
-		const Variant *p[2] = { &v, &p_data };
-		Callable::CallError ce;
-		Variant ret = get_script_instance()->call(SceneStringNames::get_singleton()->_can_drop_data, p, 2, ce);
-		if (ce.error == Callable::CallError::CALL_OK) {
-			return ret;
-		}
+	bool ret;
```

Comment:
I was adding a bunch of suggestions to initialize variables of primitive types in these cases of `GDVIRTUAL_CALL`, but eventually I've decided to leave them as they are. I'll try to make a PR that allows this declare-then-call code idiom to be more conscise.